themes/socialbase/components/00-config/variables/_forms.scss

```diff
@@ -5,7 +5,7 @@ $input-height-base:              ($line-height-computed + ($padding-base-vertica
 $form-control-max-width:         23rem;
 
 // Color help texts that go with form-controls
-$help-block:                     $gray-light;
+$help-block:                     #464545;
```

Comment:
Can we move this color into a variable to describe what it's used for? Perhaps it needs to be used elsewhere too? That might also help indicate how this color was chosen over `$gray-light`.

Reply:
I don't really sure if we need this. This is very hard to explain when you can use this variable, because this is related to background color. I don't think this will be used somewhere else. If you really want - we can create variable with the color name (http://chir.ag/projects/name-that-color/).